templates/checkout_cart.php

```diff
@@ -89,7 +89,7 @@
 
 		<?php if( has_action( 'edd_cart_footer_buttons' ) ) : ?>
 			<tr class="edd_cart_footer_row<?php if ( edd_is_cart_saving_disabled() ) { echo ' edd-no-js'; } ?>">
-				<th colspan="<?php echo edd_checkout_cart_columns(); ?>">
+				<th colspan="<?php echo esc_attr( edd_checkout_cart_columns() ); ?>">
```

Comment:
Since we're doing a `colspan` here, couldn't se just use `absint`? This would ensure we're always using a numeric value here which is a requirement of `colspan`